src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int figuresNumber = 5;
+    private static final double sideSize = 25.0;

```

Comment:
constants naming convention says it should be all capitals figuresNumber -> NUMBER_OF_FIGURE, etc